src/integrations/admin/indexation-integration.php

```diff
@@ -150,6 +151,9 @@ public function enqueue_scripts() {
 		if ( $this->is_indexation_warning_hidden() === false ) {
 			\add_action( 'admin_notices', [ $this, 'render_indexation_warning' ], 10 );
 		}
+		elseif ( $this->term_indexation->get_total_term_permalinks_null() > $shutdown_limit ) {
+			\add_action( 'admin_notices', [ $this, 'render_indexation_permalink_warning' ], 10 );
```

Comment:
You don't have to pass `10` as argument, this is the default in WordPress already. 

_It not that important to have it fixed now._